packages/cli/src/commands/init/errors/ReservedNameError.ts

```diff
@@ -1,7 +1,7 @@
 export default class ReservedNameError extends Error {
-  constructor() {
+  constructor(name: string = 'React') {
```

Comment:
TBH I would remove this default value, as it's a bit misleading given the error is not `ReactReservedNameError`. 

While I greatly appreciate your backward-compatible approach here, I think we can safely remove that one. Typescript will warn us when we don't pass argument to a constructor.